examples/ecs/iter_combinations.rs

```diff
@@ -169,8 +169,7 @@ fn integrate(mut query: Query<(&mut Acceleration, &mut Transform, &mut LastPos)>
         // verlet integration
         // x(t+dt) = 2x(t) - x(t-dt) + a(t)dt^2 + O(dt^4)
 
-        let new_pos =
-            transform.translation + transform.translation - last_pos.0 + acceleration.0 * dt_sq;
+        let new_pos = transform.translation * 2.0 - last_pos.0 + acceleration.0 * dt_sq;
```

Comment:
can you move the 2.0 to before translation to match the equation above?

Reply:
I can't do this, because we don't have the `impl Mul<Vec3> for f32` rule.